suzieq/poller/controller/source/ansible.py

```diff
@@ -108,15 +108,7 @@ def _get_inventory(self) -> Dict:
             if 'ansible_password' in entry:
                 password = entry['ansible_password']
 
-            # Retrieve password information
-            devtype = None
-            if entry.get('ansible_network_os') in ['eos', 'panos']:
-                devtype = entry.get('ansible_network_os')
-                transport = 'https'
-                port = 443
-            else:
-                transport = 'ssh'
-                port = entry.get('ansible_port', 22)
```

Comment:
If we remove this we won't be able to use the ansible inventories with panos devices. Can we leave it as is but checking only for `ansible_network_os` == `panos`?

Reply:
I had removed it for other reasons, but I think I agree with you. Adding it back in for panos.